### DISCUSSION TITLE
Random clips

### DISCUSSION THREAD

**Discussion by K1NDER-ai** [davidteather/TikTok-Api#541] [Q&A]
I wrote a system to exclude duplicate tiktok clips, after detecting a duplicate, I call trending = api.trending(3) again, but it again offers me the same video, how to make a random clip selection

**Comment by davidteather** [comment]
You can try to set a custom_did that's static across all your calls to the trending method. But beyond this, I don't know if any ways to ensure you don't get duplicates back. Your best bet is increasing the count to a big number.
